lib/components/DynamicFonts/FontDownloader.ts

```diff
@@ -97,26 +97,38 @@ export default class FontDownloader {
     fontName: string,
     fontExtension: FontExtension,
     timeout: number): Promise<string> {
-    const {downloadErrorMessage} = this.props;
+    const {downloadErrorMessage, debug} = this.props;
     await this.createPrivateFolderIfNeeded();
     const fileName = this.getFileName(fontName, fontExtension);
     const downloadLocation = this.getPrivateFilePath(fileName);
 
     try {
+      if (debug) {
+        console.log(fontName, 'Starting to download');
+      }
       const result = await this.fs.downloadFile(this.getDownloadFontOptions(fontUri, downloadLocation, timeout))
         .promise;
       if (result.statusCode === 200) {
+        if (debug) {
```

Comment:
I would still move it to another function that checks if debug is true and logs fontName, massage and other data

Reply:
Created a `log` function in each file, not ideal but I don't want to create a util for this at this time